src/PactNet.Native/IMessageMockServer.cs

```diff
@@ -0,0 +1,103 @@
+namespace PactNet.Native
+{
+    /// <summary>
+    /// Mock server
+    /// </summary>
+    internal interface IMessageMockServer
```

Comment:
Oh - is this the messaging version of the above interface? Were the methods above supposed to be deleted?

Reply:
Exactly. Each interaction model will add their own set of methods

Reply:
Can't remember what this comment was referring to but I segregated the interfaces for messaging and non messaging so all the methods that are supposed to be for messaging are carried by this interface. They were duplicated in the IMockServer interface so I removed them.